### PR TITLE
React: Different suggestion when one create suggestion

### PR DESCRIPTION
**Pull request by xitronix**
@alexcherman Could you style suggestion like in #1125 pull request?

### CODE REVIEW

**Comment by xitronix** [comment]
image

**Comment by xitronix** [comment]
image

**Comment by xitronix** [comment]
image

**Comment by xitronix** [comment]
image

File: universal-login-wallet/test/integration/pages/LoginPage.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export default class LoginPage {
   async pickUsername(userName: string, action: string, result: string) {
     const input = this.wrapper.find('input');
     input.simulate('change', {target: {value: userName}});
-    await waitForUI(this.wrapper, () => this.wrapper.text().includes(action));
+    await waitForUI(this.wrapper, () => this.wrapper.exists(`#${getSuggestionId(action)}`));
```
Comment (marekkirejczyk) [inline universal-login-wallet/test/integration/pages/LoginPage.ts]: extract function

File: universal-login-react/src/ui/WalletSelector/Suggestions.tsx [hunk 2]
```diff
@@ -8,26 +8,44 @@ interface SuggestionsProps {
...
+          Create New Account
+        </button>
+      </>
+    );
+  } else {
+    return (
+      <button className="suggestions-item-btn" id={getSuggestionId(operationType)} onClick={() => onClick(suggestion)}>
+        <p className="suggestions-item-text">{suggestion}</p>
+        <p className="suggestions-item-btn-text">{operationType}</p>
+      </button>
```
Comment (marekkirejczyk) [inline universal-login-react/src/ui/WalletSelector/Suggestions.tsx]: Extract application logic:
```ts
enum SuggestionType {
  TooShort, SingleCreation, SingleConnection, Mulitple
}
getSuggestionType(input, connections, creations): SuggestionType;
```

Component hierarchy:
```ts
Suggestion
  KeepTypingSuggestion
  SingleCreationSuggestion
  SingleConnectionSuggestion (future PR, also incude recovery link)
  MultipleOptionsSuggestion
    CreateItem
    ConnectItem
```

File: universal-login-react/src/core/models/SuggestionType.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+export type SuggestionType = 'TooShort' | 'SingleCreation' | 'SingleConnection' | 'Multiple' | 'KeepTyping';
```
Comment (marekkirejczyk) [inline universal-login-react/src/core/models/SuggestionType.ts]: Missing state: `Invalid`.

File: universal-login-react/src/core/utils/getSuggestionType.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+};
+
+const isSingleCreation = (creations: string[], connections: string[]) =>
+  creations.length === 1 && connections.length === 0;
+
+const isSingleConnection = (creations: string[], connections: string[]) =>
+  creations.length === 0 && connections.length === 1;
+
+const isKeepTyping = (creations: string[], connections: string[]) =>
+  creations.length === 0 && connections.length === 0;
```
Comment (marekkirejczyk) [inline universal-login-react/src/core/utils/getSuggestionType.ts]: In this case, might be too short or invalid.

**Pull request merged**